### DISCUSSION TITLE
Watchstate from all managed Plex users being pulled in instead of certain accounts...

### DISCUSSION THREAD

**Discussion by almightiest** [arabcoders/watchstate#295] [General] [answered]
Ideally, I'd pull in watched status from only locally managed accounts for my wife and I into watchstate.  If I can only do a single account, that's fine I'll live with that.  My main problem right now is that for a LONG time I could never pinpoint why randomly some things were getting marked as watched and I finally found out it was tied to managed users in Plex having their watch events get imported, I think through webhooks. I was looking at Plex play history today and saw that as managed users were watching things, at the top of the next hour it was marking those items as watched for my local admin account. I'm trying to figure out how to narrow this down to import for only a specific set of accounts.  I can't figure it out in the docs alone, and here's what I've tried so far:

1. Get list of plex users.  I am userid 1 (admin acct), and that's the main account we use to watch.  My wife has a local account too but mainly uses the default (she's fine with her stuff not syncing).
2. Inspect the Token and jot it down
3. View the config for plex
4. Observe that match user = false and match uuid = false
5. Change the config to set those to true
6. Observe that the tokens do not match what I saw on the admin user
7. Create this discussion since I'm stuck on next steps :-D

**Comment by arabcoders** [comment, marked as answer]
Hi, 

Great questions, Let me try to answer them.

> Observe that the tokens do not match what I saw on the admin user

That's expected as we request new token using the provided admin token, as this token sometimes is temporary. nothing to worry about. as long as you get your data synced it doesn't matter which token shows in config.

> My main problem right now is that for a LONG time I could never pinpoint why randomly some things were getting marked as watched and I finally found out it was tied to managed users in Plex having their watch events get imported, I think through webhooks.

This should not happen automatically, unless you actually enabled webhooks and forgot to set match user, plex way of doing webhook's is kinda not ideal as they used the endpoint for all users regardless as such if match user is off, it's expected that the event be recorded.

What you should is start clean add your plex backend, make sure to enable both match user id and backend id, add the webhook endpoint to your plex server, and observe the logs if it's says the user id incorrect or backend id is incorrect, it should also tell you what value was provided by the user you can then change it to match your user.

Plex way of doing webhook's is really unhelpful and sometimes downright wrong, user id changing depending on having managed users or not is not something that i can account for =)

Anyway, Enable both options i listed and check the logs and link them if you cant proceed further.
